### PR TITLE
Use auto_lang mode to detect if lang is forced or not

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
Solves the problem that code.language doesn't update when forcing a language and until you change focus and the app title updates you have 2 language context active at the same time.

This is active if user.javascript OR code.language == javascript.
```
mode: user.javascript
mode: command
and code.language: javascript
```

Change it to this and the problem is solved sine auto_lang and user.javascript is mutex.
```
mode: user.javascript
mode: user.auto_lang
and code.language: javascript
```

### CODE REVIEW

File: code/code.py [hunk 1]
```diff
@@ -87,11 +84,9 @@ def code_libraries(m) -> str:
 class code_actions:
     def language():
         result = ""
-        if not forced_language:
-            file_extension = actions.win.file_ext()
-
-            if file_extension and file_extension in extension_lang_map:
-                result = extension_lang_map[file_extension]
+        file_extension = actions.win.file_ext()
```
Comment (knausj85) [inline code/code.py:87]: I feel like there's scenarios where both auto_lang and force lang could be active with this as-is. Maybe worth checking the auto_lang mode here for sanity?

Comment (AndreasArvidsson) [reply code/code.py:87]: In which scenario are you thinking about?
As the code is written now you disable auto_lang when a specific language mode is forced/enabled and you enable auto_mode when forced language is cleared/disabled. So they should be mutually exclusive.

Comment (knausj85) [reply code/code.py:87]: Sorry, you're correct. Handled by the context checks.

```
mode: user.auto_lang
and code.language: javascript
```

Comment (AndreasArvidsson) [reply code/code.py:87]: Yes exactly. As far as I can tell this should work.

**Review by knausj85 — APPROVED** [review]

**Pull request merged**